### PR TITLE
Elastic beat upgrade

### PR DESCRIPTION
**Pull request by vishalxebia**
1.) added pid in connbeat output.
2.) upgrade the elastic beat to 5.6.9

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,6 @@ BEAT_PATH=github.com/raboof/connbeat
 SYSTEM_TESTS=true
 TEST_ENVIRONMENT?=true
 ES_BEATS?=./vendor/github.com/elastic/beats
-BEAT_GOPATH=/vagrant/gocode
```
Comment (raboof) [inline Makefile]: Was this necessary?

**Review by raboof — COMMENTED** [review]
One question, otherwise OK if you have tested that the project still works well with the new version of beat

File: Makefile (hunk 1 above)
Comment (vishalxebia) [reply Makefile]: No it was just a trial to check a scenario but realized later it wasn't needed.

**Comment by raboof** [comment]
Thanks!

**Pull request merged**